src/ploneorg/releasesecurityinfo/field.py

```diff
@@ -0,0 +1,45 @@
+from hashlib import sha1, md5
+from zope.interface import implements
+from zope import schema
+from zope.schema.fieldproperty import FieldProperty
+from plone.namedfile.interfaces import INamedFile, INamedFileField
+from plone.namedfile.field import NamedFile
+from plone.namedfile.file import NamedFile as FileValueType
+
+
+class IChecksummedFile(INamedFile):
```

Comment:
didn't we spoke that no file field is needed? why did you add those File field elements?